include/tvm/runtime/contrib/dnnl/dnnl_kernel.h

```diff
@@ -0,0 +1,57 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+/*!
+ * \file include/tvm/runtime/contrib/dnnl/dnnl_kernel.h
+ * \brief Use external dnnl library kernels.
+ */
+
+#ifndef TVM_RUNTIME_CONTRIB_DNNL_DNNL_KERNEL_H_
+#define TVM_RUNTIME_CONTRIB_DNNL_DNNL_KERNEL_H_
+
+#include "dnnl.hpp"
+
+namespace tvm {
+namespace runtime {
```

Comment:
The header should be part of the internal header, (move to src/) as we don't want to expose them to users' of tvm

Reply:
Yeah, I intentionally moved it from src to header. The reason was because the wrapper could then just directly include it and `export_libaray` will then take care of finding the path under `include/tvm` here:

https://github.com/apache/incubator-tvm/blob/279a8ebae6d507f02d904397672dc44982719645/python/tvm/_ffi/libinfo.py#L183

Otherwise, We may expect users to pass `-I$PATH_TO_TVM/src/runtime/contrib`